docs/components/panels/index.html

```diff
@@ -4,19 +4,17 @@
 {% extends 'component.njk' %}
 {% block content %}
   <section>
-    <h2 class="hxSectionTitle" id="summary">Summary</h2>
     <p>
       A panel is a utility for sectioning content vertically into a head,
```

Comment:
Do you mention that the head and foot are optional anywhere other than on the component page?

Reply:
Panel component and `<hx-panel>` element documents updated to mention the head and foot are optional